eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/FeeProvider.scala

```diff
@@ -136,8 +136,8 @@ object FeeratesPerKw {
     fastest = FeeratePerKw(feerates.fastest))
 
   /** Used in tests */
-  def single(feeratePerKw: FeeratePerKw): FeeratesPerKw = FeeratesPerKw(
-    minimum = feeratePerKw,
+  def single(feeratePerKw: FeeratePerKw, networkMinFee: FeeratePerKw = FeeratePerKw(1.sat)): FeeratesPerKw = FeeratesPerKw(
```

Comment:
nit: 1 sat/kw doesn't really make sense, you can't go lower than 1 sat/byte.

```suggestion
  def single(feeratePerKw: FeeratePerKw, networkMinFee: FeeratePerKw = FeeratePerKw(FeeratePerByte(1 sat))): FeeratesPerKw = FeeratesPerKw(
```